.github/workflows/CI.yml

```
@@ -0,0 +1,50 @@
+on:
+  pull_request:
```

```suggestion
  push:
    branches:
      - master
  pull_request:
```

see. https://zenn.dev/ikawaha/articles/055cc7070ff0d12c5b10

---

Fixed.